src/test/rustdoc/source-version-separator.rs

```diff
@@ -0,0 +1,31 @@
+#![stable(feature = "bar", since = "1.0")]
+#![crate_name = "foo"]
+
+#![feature(staged_api)]
+
+// @has foo/trait.Bar.html
+// @has - '//div[@class="main-heading"]/*[@class="out-of-band"]' '1.0· source · '
```

Comment:
There's a missing space here.

```suggestion
// @has - '//div[@class="main-heading"]/*[@class="out-of-band"]' '1.0· source · '
```

Reply:
@jsha I think you forgot to add the space in your suggestion.

Reply:
Same answer as above.